### PR TITLE
Update docs to include app related config changes for vizio integration

### PR DESCRIPTION
**Pull request by raman325**
## Proposed change
I have submitted a PR to support app launching to the vizio integration. Along with those changes, I have introduced new config parameters which can be used to tweak the behavior of the app support.

Also noticed a couple of things weren't following standards while reviewing the developer docs on documentation so updated those

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [x] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: home-assistant/core#32432

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/vizio.markdown [hunk 1]
```diff
@@ -21,7 +21,11 @@ If `zeroconf` discovery is enabled, your device will get discovered automaticall
 
 ### Install `pyvizio` locally
 
-> NOTE: If the `pip3` command is not found, try `pip` instead
+<div class='note tip'>
```
Comment (frenck) [inline source/_integrations/vizio.markdown]: `tip` is not a valid class
```suggestion
<div class='note'>
```

File: source/_integrations/vizio.markdown [hunk 2]
```diff
@@ -130,8 +134,64 @@ volume_step:
...
+              required: true
+              type: integer
+            MESSAGE:
+              description: See [Obtaining an app configuration](#obtaining-an-app-configuration) section below.
+              required: false
+              type: string
+              default: null
 {% endconfiguration %}
 
+### Obtaining an app configuration
```
Comment (frenck) [inline source/_integrations/vizio.markdown:179]:
```suggestion
### Obtaining an app configuration

```

**Review by frenck — CHANGES_REQUESTED** [review]
2 small suggestions. Could you take a look? 👍

**Comment by raman325** [comment]
thanks @frenck committed both of your suggestions. I did notice `tip` being used in other places along with `info`

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Pull request merged**